### DISCUSSION TITLE
Animation Concerns

### DISCUSSION THREAD

**Discussion by hackyfemmyboi** [ManimCommunity/manim#2260] [General]
I'd like to send my concersn concerning manim here. First off, HOW THE HECK DO I ANIMATE 3B1B'S PI CREATURES??? I tried everything, but still nothing!!

**Comment by Darylgolden** [comment]
I know you're probably trying to adopt a playful/very informal rather than hostile tone, but this message and the messages in the previous discussion come across as quite rude and unprofessional. Please adopt a friendly and considerate tone instead.

As for your question, 3blue1brown's pi creatures are copyrighted by him and only intended to be used in his videos, which is why they're deliberately omitted from this library. You should not attempt to use them without his explicit permission.

**Reply by hackyfemmyboi** [reply]
shucks. sorry for the forceful tone

**Reply by hackyfemmyboi** [reply]
It... just drove me a bit crazy

**Reply by hackyfemmyboi** [reply]
hmmm

**Reply by hackyfemmyboi** [reply]
how do I change the color of the complex plane?
